content/applications/general/users/access_rights.rst

```diff
@@ -52,6 +52,65 @@ The :guilabel:`Administration` field in the :guilabel:`Access Rights` tab has th
 .. image:: access_rights/user-permissions-dropdown-menu.png
    :alt: The Sales apps drop-down menu to set the user's level of permissions.
 
+Manage specific permissions
+---------------------------
+
+While access rights are typically assigned in bundles under specific roles, they can also be set as
+explicit permissions.
+
+.. example::
+   For example, giving a user the :guilabel:`Administrator` permission for **Timesheets**
+   gives them full access to that app. That user, while holding full access, can *still* have their
+   ability to manage *their own* timesheets restricted — such as in the case of a salaried payroll
+   administrator who does not need to track time.
+
+To manage specific permissions, :ref:`developer mode <developer-mode>` must be enabled. After that,
+navigate to the :menuselection:`Settings` app. Then click :guilabel:`Manage Users`, select a user,
+and go to the :guilabel:`Technical Access Rights` tab. From here, :guilabel:`Groups` can be edited,
+and specific access rights can be managed across the various sections. If no changes are made to
+these *Groups*, then their permissions will mirror the selections made in the :guilabel:`Access
+Rights` tab.
+
+- :guilabel:`Selected groups`: a list of detailed access rights, set by choices made in the
+  :guilabel:`Access Rights` tab.
+- :guilabel:`Groups added automatically`: *implied* permissions that are *inherited* with the
+  explicit permissions already granted to the user. The values here will match the values listed
+  under a given *Group*'s form located under the :menuselection:` Users & Companies --> Groups`
+  menu, in the :guilabel:`Inherited` tab.
+
+.. image:: access_rights/tech-access-rights.png
+   :alt: The technical access rights tab opened up for a user profile.
+
+.. example::
+   When the *Sales Administrator* permission set is assigned to a user, then the *Canned Responses
+   Administrator* permissions are inherited automatically. These assignments are reflected across
+   the values listed in the :guilabel:`Selected Groups` and :guilabel:`Groups added automatically`
+   tables, respectively.
+
+To add a permission to this user profile, click :guilabel:`Add a line` in the :guilabel:`Selected
+groups` table, and then add permissions to this user profile. To remove a permission, click the
+:icon:`fa-times` :guilabel:`(cancel)` at the end of that permission's row.
+
+.. warning::
+   Removing permissions from the *Selected groups* list can impact what permissions are listed in
+   the *Groups added automatically* list, since selected permission groups inform what permission
+   groups are added automatically.
```

Comment:
UI text should be formatted in guilabel when part of current UI
```suggestion
   Removing permissions from the :guilabel:`Selected Groups` list can impact what permissions are
   listed in the :guilabel:`Groups added automatically` list, since selected permission groups
   inform what permission groups are added automatically.
```

Reply:
Also optional, it's part of a hypothetical warning not an active instruction